shared/ce/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@fmc/ce",
+  "description": "Stencil Component Starter",
+  "main": "dist/index.cjs.js",
+  "module": "dist/index.js",
+  "es2015": "dist/esm/index.js",
+  "es2017": "dist/esm/index.js",
+  "types": "dist/types/index.d.ts",
+  "collection": "dist/collection/collection-manifest.json",
+  "collection:main": "dist/collection/index.js",
+  "unpkg": "dist/fmc-ce/fmc-ce.esm.js",
+  "files": [
+    "dist/",
+    "loader/"
+  ],
+  "scripts": {
+    "build": "stencil build --docs",
+    "start": "stencil build --dev --watch --serve",
```

Comment:
Please check if addition of this serves the purpose

`"dev": "stencil build --watch --serve"`

Reply:
Yes its working.
Should I keep the `--serve`? Because it is launching the localhost by itself. If we just want use the custom-elements, then we don't need to open the localhost ?

Reply:
Lets get rid of it.